pom.xml

```diff
@@ -125,8 +125,8 @@
         <javax.json.version>1.1.4</javax.json.version>
         <rest-assured.version>4.4.0</rest-assured.version>
         <rest-assured-json-path.version>4.4.0</rest-assured-json-path.version>
-        <jupiter.version>5.7.0</jupiter.version>
-        <junit.platform.version>1.7.0</junit.platform.version>
+        <jupiter.version>5.8.2</jupiter.version>
```

Comment:
I will rebase https://github.com/strimzi/strimzi-kafka-operator/pull/6416 (when it will be merged)